tools/cp-defs.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs')
+const path = require('path')
+
+const src = path.resolve(__dirname, '../electron.d.ts')
+const target = process.argv[2]
+
+if (!target) return
+
+fs.writeFileSync(target, fs.readFileSync(src))
```

Comment:
it seems like you could just do this with GN's `copy` action, or just pass an extra `--out` to `npm run create-typescript-definitions`